yb_build.sh

```diff
@@ -461,6 +461,10 @@ run_cxx_build() {
     if is_mac && [[ "${YB_TARGET_ARCH:-}" == "arm64" ]]; then
       cmake_binary=/opt/homebrew/bin/cmake
     else
+      if ! which cmake &> /dev/null; then
+          echo "Error: cmake not found in PATH" >&2
+          exit 1
```

Comment:
```suggestion
        echo "Error: cmake not found in PATH" >&2
        exit 1
```

Reply:
No worries, looking forward to making more contributions! Love all the value yugabyte provides :)